e2e_test/batch/keyword/user.slt

```diff
@@ -0,0 +1,16 @@
+statement ok
+SET RW_IMPLICIT_FLUSH TO true;
+
+statement ok
+create table t (id int, user varchar);
```

Comment:
In PostgreSQL it would has to be `"user" varchar` because `user` is a reserved keyword. What about also using the preferred syntax here and acknowledge in comments that we accidentally also allow unquoted `user` as column name in DDL, but in SELECT it would have to be quoted?

Reply:
We have already allowed users to use `user` as a column name, and it seems to have no negative effect because we also allow users to use `"user"` as a column name.

Reply:
We shall encourage usage of more standard syntax.

Reply:
We can use another breaking change PR to do so if necessary.🤔